src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -5157,6 +5255,283 @@ namespace AirflowNetworkBalanceManager {
 		return CalcWindPressure;
 	}
 
+	Real64
+	AirDensity(
```

Comment:
Not sure what @jasondegraw intended, but I would always use an existing Psy function instead of introducing a new one.